chrysom/basicClient.go

```diff
@@ -52,18 +55,18 @@ type BasicClientConfig struct {
 	// Bucket partition to be used by this client.
 	Bucket string
 
-	// HTTPClient refers to the client that will be used to send requests.
-	// (Optional) Defaults to http.DefaultClient.
-	HTTPClient *http.Client
-
 	// Auth provides the mechanism to add auth headers to outgoing requests.
 	// (Optional) If not provided, no auth headers are added.
 	Auth Acquirer
```

Comment:
The http.Client and Acquirer should be replaced by a the `arrangehttp.ClientConfig`.  I think this lets you delete most of chrysom.

Reply:
I was thinking about using `arrangehttp.ClientConfig`, but I saw more trouble than it's worth (potentially having ancla's arrange version clash with other servers versions).

I was thinking having the user provide a http.client was a the amplifier path forward.